checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitTransfer.java

```diff
@@ -361,9 +361,12 @@ public TransferResult<NullnessNoInitValue, NullnessNoInitStore> visitMethodAcces
             MethodAccessNode n, TransferInput<NullnessNoInitValue, NullnessNoInitStore> p) {
         TransferResult<NullnessNoInitValue, NullnessNoInitStore> result =
                 super.visitMethodAccess(n, p);
-        // In contrast to the conditional makeNonNull in visitMethodInvocation, this
-        // makeNonNull is unconditional, as the receiver is definitely non-null after the access.
-        makeNonNull(result, n.getReceiver());
+        // In contrast to the makeNonNull in visitMethodInvocation, which checks the method is
```

Comment:
This comment is rather complicated and inconsistent - before, this was unconditional and the other one was conditional; that distinction no longer exists, so the "in contrast" makes no sense.

How about: `The receiver of an instance method access is non-null. A static method access does not ensure that the receiver is non-null.`

Reply:
Ok. I will add change to this sentence.